products-service/src/functions/getAllProducts/index.ts

```diff
@@ -0,0 +1,13 @@
+import { handlerPath } from '@libs/handler-resolver';
+
```

Comment:
are you going to add swagger, or no? just want to clarify. Thanks.

Reply:
No for now.